docs/install-mender.md

```diff
@@ -122,22 +123,47 @@ This method requires appending `local.conf` with necessary recipes by using the
 
 ```
 IMAGE_INSTALL:append = " \ 
-    thin-edge \
     tedge \ 
-    tedge-mapper \
-    tedge-agent \
-    c8y-configuration-plugin \
-    c8y-log-plugin \
-    tedge-apt-plugin \
-    tedge-dummy-plugin \
-    tedge-watchdog \
+    tedge-bootstrap \
+    tedge-inventory \
     tedge-state-scripts \
+    tedge-firmware
 "
 ```
-Then you can use any image delivered by the Yocto team. Thin-edge requires at least `core-image-minimal` to function correctly:
+
+You must set fixed uid/gid to avoid permissions problems on the `/data` partition. You can do that by inheriting the `extrausers` class in your image recipe or the `local.conf` file:
+
+```
+inherit extrausers
+EXTRA_USERS_PARAMS = "\
+    groupmod -g 960 mosquitto; \
+    usermod -u 961 mosquitto; \
+"
+```
```

Comment:
question: Is this something we can't do in our recipes?

Reply:
yeah I will try to address that in future PR.